Jenkinsfile

```diff
@@ -73,6 +84,17 @@ pipeline {
             sh './gradlew --no-daemon itest'
           }
         }
+        stage('ce itest jdk12') {
```

Comment:
I think we do not have to check all variant, as we test jdk12 already lets skip this